packages/remix-dev/__tests__/cli-test.ts

```diff
@@ -133,6 +134,21 @@ describe("remix cli", () => {
   });
 
   describe("the create command", () => {
+    afterAll(() => {
+      async function renamePkgJsonApp(dir: string) {
+        let pkgPath = path.join(dir, "package.json");
+        let pkg = await fsp.readFile(pkgPath);
+        let obj = JSON.parse(pkg.toString());
+        obj.name = path.basename(dir);
+        await fsp.writeFile(pkgPath, JSON.stringify(obj, null, 2) + "\n");
+      }
+
+      let dirs = fs.readdirSync(path.join(process.cwd(), ".tmp"));
+      for (let dir of dirs) {
+        renamePkgJsonApp(path.join(process.cwd(), ".tmp", dir));
+      }
```

Comment:
Why do we need to change the `name` in `package.json` for each of the created apps?

Reply:
jest-haste https://github.com/remix-run/remix/runs/5579325183?check_suite_focus=true#step:7:6

Reply:
Does the naming collision happen if we just omit the `name` field from the apps?

Since these are apps and not libraries, the `name` field is misleading anyway.

Reply:
let me see, that'd be great if we could omit them :)

we'd just need to remove them from the examples/templates directories as well

Reply:
so far it seems fine, gonna run it a few times to pile up the amount of apps in .tmp and open a PR to remove all our shenanigans